arm/Microsoft.EventHub/namespaces/deploy.bicep

```diff
@@ -43,8 +43,12 @@ param authorizationRules array = [
 @description('Optional. Configuration Details for private endpoints.For security reasons, it is recommended to use private endpoints whenever possible.')
 param privateEndpoints array = []
 
-@description('Optional. Networks ACLs, this value contains IPs to whitelist and/or Subnet information. For security reasons, it is recommended to set the DefaultAction Deny.')
-param networkAcls object = {}
+@description('Optional. Networks ACLs, this object contains IPs/Subnets to whitelist or restrict access to private endpoints only. For security reasons, it is recommended to configure this object on the Namespace.')
+param networkRuleSets object = {
```

Comment:
I wonder if those values are considered `secure` - or if (by default) e.g. `publicNetworkAccess` should be disabled. We should probably compare this with the corresponding security baseline

Reply:
I checked this with some resources in the library that use the same `publicNetworkAccess` property. 
- Container Registry = enabled
- Data factory = enabled
- Storage Account = enabled
- Key Vault = enabled
- Event Grid = enabled
- Log Analytics = enabled
- Machine learning = disabled
- disks = disabled

As most have it `enabled` by default. Setting this to `disabled` only allows access via private endpoints. And given there is another mechanism here is to keep this to `enabled` and still use the PaaS specific Firewall to allow service endpoints / IP whitelisting. I would suggest that if we are to enforce private endpoints only on carml resources, we roll it out via a separate issue to discuss the approach and implement this across the board. Thoughts?

Reply:
I agree that disabling network access by default would only make sense if we also enabled private endpoints by default which as per recent discussion we decided not to do.